R/cubist_rules.R

```diff
@@ -180,8 +180,11 @@ tunable.cubist_rules <- function(x, ...) {
 
 #' `multi_predict()` methods for rule-based models
 #' @rdname multi_predict
-#' @export
+#' @param object An object of class `model_fit`
+#' @param new_data A rectangular data object, such as a data frame.
+#' @param type A single character value or `NULL`.
```

Comment:
Am I correct that `type` is not passed anywhere in this method? Changed this in previous commit, but we could overwrite the inherited `param` here to note that `type` is not passed anywhere.

Reply:
Other functions will pass `type` in (I believe that happens in tune). There is only one type for this method and the internal `cubist_pred()` handles that.

Reply:
Got it! Okay, will add on more commit to clarify docs.